mshv-ioctls/src/ioctls/vm.rs

```diff
@@ -88,7 +88,7 @@ impl VmFd {
         &self,
         interrupt_type: hv_interrupt_type,
         apic_id: u64,
-        vector: u32,
+        _vector: u32,
```

Comment:
Why is this needed? `vector` is used.

Reply:
Giving the chippy warnings. with the same name

Reply:
Does it go away after adopting my suggestion below?